Sources/Prepare.swift

```diff
@@ -50,7 +50,14 @@ public struct Prepare: Command {
 
             let hasPrepared = try database.hasPrepared(preparation)
             // only prepare the unprepared
-            guard !hasPrepared else { return }
+            guard !hasPrepared else { 
+                if let model = preparation as? Entity.Type {
```

Comment:
this seems unexpected in the guard, should the database be set either way? Otherwise a quick comment about why we have to do it here.